src/index.html

```diff
@@ -15,8 +15,96 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list__item">
+            <a
+              class="nav__list__item__link is-active"
+              href="#"
+            >
```

Comment:
The class names should represent the meaning of the content, not the structure or style. Instead of `nav__list__item__link`, consider using a simpler name like `nav__link`. This applies to all occurrences of `nav__list__item` and `nav__list__item__link`.